cmd/ib-sriov-cni/main.go

```diff
@@ -164,15 +160,82 @@ func cmdAdd(args *skel.CmdArgs) error {
 		}()
 	}
 
-	result := &current.Result{}
-	result.Interfaces = []*current.Interface{{
-		Name:    args.IfName,
-		Sandbox: netns.Path(),
-	}}
-
 	err = sm.SetupVF(netConf, args.IfName, args.ContainerID, netns)
+	if err != nil {
+		return fmt.Errorf("failed to set up pod interface %q from the device %q: %v", args.IfName, netConf.Master, err)
```

Comment:
why netconf.Master and not netConf.DeviceID ?

the "Master" is the PF netdevice, is that what you want to mention in this error message ? if so then why ? 

SetupVF method doesnt use the PF.

Reply:
i see this line was simply moved, but lets change netConf.Master to netConf.DeviceID :) WDYT ?